data/map.txt

```diff
@@ -27384,6 +27532,15 @@ planet "Graveyard Wormhole (Unstable)"
 planet "Graveyard Wormhole (Stable)"
 	description ``
 
+planet Greymoon
+	attributes station hai
+	landscape land/space6
+	description `This station is owned and operated by the Far Loo Bah fuel collection corporation. This is a fact that you cannot avoid seeing, as their logo is emblazoned on nearly every possible surface and sign indicating their sponsorship of every service or feature on the station. Interestingly this includes a "History of Spaceflight" museum which takes up an entire branch of the station. Naturally, the museum focuses very heavily on the Far Loo Bah corporation's role in that history.`
+	spaceport `The spaceport consists of a wide two-level promenade encircling the interior of the station, with the docking access opening directly onto the mezzanine. The upper level is mostly devoted to amenities, light commercial businesses, a medical center, and a bank. The lower level has all of the real businesses, including frontage from multiple manufacturers supplied by the heavier elements filtered out from the supply of fuel stock from the planet below. A virtual outfitter provides sale options for locally-manufactured outfits that can be installed on your ship without you ever leaving the comfort of the promenade.`
```

Comment:
```suggestion
	spaceport `The spaceport consists of a wide, two-level promenade encircling the interior of the station, with the docking access opening directly onto the mezzanine. The upper level is mostly devoted to amenities, light commercial businesses, a medical center, and a bank. The lower level has all of the real businesses, including frontage from multiple manufacturers supplied by the heavier elements filtered out from the supply of fuel stock from the planet below. A virtual outfitter provides sale options for locally-manufactured outfits that can be installed on your ship without you ever leaving the comfort of the promenade.`
```
I think there's room for improvement on `including frontage from multiple manufacturers supplied by the heavier elements filtered out from the supply of fuel stock from the planet below`. I can't really tell what's being explained, but maybe it's just me. Is there a way to make this more concise?

Reply:
The idea is that they're siphoning gasses from the planet below to make fuel, and that it being a large gas giant it has other heavier elements whipped up as dust in its atmosphere (from it's planet-sized storms) which have to be filtered out and are provided to the manufacturers on-station.

Reply:
Ah, thanks. How about this?
```suggestion
	spaceport `The spaceport consists of a wide, two-level promenade encircling the interior of the station, with the docking access opening directly onto the mezzanine. The upper level is mostly devoted to amenities, light commercial businesses, a medical center, and a bank. The lower level has all of the real businesses: multiple manufacturers, which are supplied by the heavier elements filtered out from the supply of fuel stock from the planet below, have store frontage here. A virtual outfitter provides sale options for locally-manufactured outfits that can be installed on your ship without you ever leaving the comfort of the promenade.`
```